### PR TITLE
[stable/elasticsearch-exporter] Fix typo - Add Readiness probe to exporter deployment.yaml

### PR DESCRIPTION
**Pull request by nithyanatarajan**
#### What this PR does / why we need it:
The `deployment.yaml` file in stable `elasticsearch-exporter` chart has two liveness probes configured, one with `initialDelaySeconds: 30` and another with `initialDelaySeconds:10`. I have modified one of the probes to readiness probe assuming that it was a miss.

#### Special notes for your reviewer:
@svenmueller Kindly have a look at this simple fix.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped

### CODE REVIEW

**Comment by NithyaNatarajan** [comment]
sure, will update and push 👍

**Review by svenmueller — APPROVED** [review]
LGTM

**Comment by svenmueller** [comment]
@NithyaNatarajan no problem :) we need to wait for an approval from a member the owners list (https://github.com/helm/charts/blob/master/OWNERS)

**Comment by NithyaNatarajan** [comment]
@svenmueller got it, thank you 🙏

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**
